src/Humanizer/DateTimeHumanizeStrategy/IDateTimeHumanizeStrategy.cs

```diff
@@ -12,5 +12,10 @@ public interface IDateTimeHumanizeStrategy
         /// Calculates the distance of time in words between two provided dates used for DateTime.Humanize
         /// </summary>
         string Humanize(DateTime input, DateTime comparisonBase, CultureInfo culture);
+
+        /// <summary>
+        /// Calculates the distance of time in words between two provided dates used for DateTime?.Humanize
+        /// </summary>
+        string Humanize(DateTime? input, DateTime comparisonBase, CultureInfo culture);
```

Comment:
I don't know whether we really need to have this method on `Strategy`.
As far as I can currently see, it would always have the same implementation:
```
if (input.HasValue)
  return Humanize(input.Value, comparisonBase, culture);
else
  return Configurator.GetFormatter(culture).DateHumanize_Never();
```

I would implement this directly in the `DateHumanizeExtensions` class.
By this the impact on the API by this feature would be minimized, and if someday there will be a need to have different strategies for `null` values, we can implement it the way currently shown here.